components/XML/Tests/XMLProcessorTest.php

```diff
@@ -214,10 +214,10 @@ public function test_malformed_attribute_value_containing_ampersand_is_treated_a
 	 * @covers XMLProcessor::get_attribute
 	 */
 	public function test_malformed_attribute_value_containing_entity_without_semicolon_is_treated_as_plaintext() {
-		$processor = XMLProcessor::create_from_string( '<wp:content enabled="&#x94">Test</wp:content>' );
+		$processor = XMLProcessor::create_from_string( '<wp:content xmlns:wp="w.org" enabled="&#x94">Test</wp:content>' );
 
 		$this->assertTrue( $processor->next_tag(), 'Querying a tag did not return true' );
-		$this->assertEquals( '&#x94', $processor->get_attribute( 'enabled' ) );
+		$this->assertEquals( '&#x94', $processor->get_attribute( '', 'enabled' ) );
```

Comment:
these are good, and the thing I wanted to review most specifically. it was confusing to me how attributes behave since they are _not_ namespaced without an explicit namespace.